packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.styles.ts

```diff
@@ -78,7 +78,7 @@ export const getArrowButtonStyles = memoizeFunction((
       }
     },
     rootDisabled: {
-      opacity: 0,
+      opacity: 0.5,
       [MS_HIGHCONTRAST_ACTIVE]: {
         color: 'GrayText'

```

Comment:
in high contrast we need to put the opacity back to 1

Reply:
I'll update PR with this before you guys merge.